lib/delayed/performable_method.rb

```diff
@@ -1,8 +1,8 @@
 module Delayed
   class PerformableMethod
-    attr_accessor :object, :method_name, :args
+    attr_accessor :object, :method_name, :args, :kwargs
 
-    def initialize(object, method_name, args)
+    def initialize(object, method_name, args, kwargs = {})
```

Comment:
@korny @NourEldinShobier We could either keep it this way of requiring args before kwargs. Or we could make args optional, i.e. `def initialize(object, method_name, *args, **kwargs)` - in that case we'll need to change some code.

Reply:
Not really. The signature should stay as it is for compatibility with older jobs. `args` is an Array and can be empty, but it can not be missing.